pkg/commands/workload.go

```diff
@@ -156,6 +161,15 @@ func (opts *WorkloadOptions) Validate(ctx context.Context) validation.FieldError
 		errs = errs.Also(validation.CompareQuantity(opts.LimitMemory, opts.RequestMemory, flags.RequestMemoryFlagName))
 	}
 
+	if opts.RegistryPassword != "" || opts.RegistryUsername != "" || opts.RegistryToken != "" {
```

Comment:
why isnt this true for CA cert paths flag?

Reply:
I missed it. Thanks for pointing it out.